examples/datadog-express-apm/container-agent/docker-compose.yml

```diff
@@ -1,13 +1,20 @@
-version: '3.5'
 services:
   app:
-    build: .
+    build:
+      context: .
+    env_file:
+      - .env
     environment:
+      - DD_SITE=${DD_SITE}
+      - DD_API_KEY=${DD_API_KEY}
+      - NODE_ENV=production
       - DD_ENV
       - DD_TRACE_DEBUG
       - DD_TRACE_ENABLED
       - DD_LOGS_INJECTION=true
       - DD_AGENT_HOST=agent
+    ports:
+      - "9000:9000"
```

Comment:
I did not try to run the docker app, but here you are exposing port 9000, and in the dockerfile you are exposing port 8080. How does it work?

Reply:
you are correct I am changing the port exposed in the dockerfile. Thank you!